apps/site/assets/ts/tnm/components/RouteCard.tsx

```diff
@@ -44,7 +44,7 @@ const RouteCard = ({
     return null;
   }
 
-  const hasAlert = route.route.alerts.length > 0;
+  const hasAlert = route.route.alerts && route.route.alerts.length > 0;
```

Comment:
nit-pick: You could use the nullish operator here `routes.route.alerts?.length > 0`